lnbits/core/services/notifications.py

```diff
@@ -288,12 +288,11 @@ async def send_ws_payment_notification(wallet: Wallet, payment: Payment):
             "payment": json.loads(payment.json()),
         },
     )
-    await websocket_manager.send_data(payment_notification, wallet.inkey)
```

Comment:
why break backwards compability?

are there definitely no extensions that use this?

Reply:
yes, the down below `websocket_updater`  was used